src/components/Search/SearchView/SearchView.tsx

```diff
@@ -27,28 +25,7 @@ function SearchView(props: SearchViewProps) {
     <Container maxWidth="lg" onClick={handleParentClick}>
       {/* Component to fetch recent revisions on mount */}
       <SearchViewInit />
-      <Box>
-        <Typography
-          variant="h1"
-          component="div"
-          align="center"
-          gutterBottom
-          sx={{
-            '&:after': {
-              backgroundImage: `url(${String(zap)})`,
-              backgroundPosition: '55%',
-              backgroundRepeat: 'no-repeat',
-              backgroundSize: '25%',
-              content: '""',
-              display: 'block',
-              height: '0.3em',
-              marginTop: '-5px',
-            },
-          }}
-        >
-          PerfCompare
-        </Typography>
-      </Box>
+      <Header />
```

Comment:
`Header` sounds a bit generic, could you maybe rename this component to something more suggestive?